### PR TITLE
ArgoCD to 1.61

### PR DESCRIPTION
**Pull request by HumairAK**
## This introduces a breaking change

- [x] Yes
- [ ] No

Updates manifests, dockerfile and ksops version (along with kustomize version). 

Also updated the playbook to be more dev friendly and removed a duplicate project deployment task.

### CODE REVIEW

**Comment by HumairAK** [comment]
Still need to verify in dev

File: objects/namespace-install.yaml [hunk 1]
```diff
@@ -1763,6 +1808,23 @@ spec:
                 - kind
                 type: object
               type: array
+            namespaceResourceWhitelist:
+              description: NamespaceResourceWhitelist contains list of whitelisted
+                namespace level resources
+              items:
+                description: GroupKind specifies a Group and a Kind, but does not
+                  force a version.  This is useful for identifying concepts during
```
Comment (anishasthana) [inline objects/namespace-install.yaml:1816]: `  This is` Extra whitespace

Comment (HumairAK) [reply objects/namespace-install.yaml:1816]: this is from upstream, I'd rather not touch it as we're going to move to kustomize anyway, you should bring it up with the argo-cd folks :)

**Comment by HumairAK** [comment]
Tested  and works in dev

**Pull request merged**